src/main/java/lotto/utils/Validation.java

```diff
@@ -10,4 +10,18 @@ public static Boolean isNumeric(String value) {
         }
         return true;
     }
+
+    public static Boolean isNumericAndPositive(String value) {
+        if (!isNumeric(value)) {
+            return false;
+        }
+
+        int valueInt = Integer.parseInt(value);
+
+        if (valueInt < 0) {
+            return false;
+        }
+
+        return true;
```

Comment:
```suggestion
        Integer.parseInt(value) >= 0;
```

boolean 반환 시에 항상 고려해주세요 :)